components/CheckBalanceAmount.jsx

```diff
@@ -131,11 +153,9 @@ export default () => {
             {filAddress} has {balance.toFil()} FIL.
           </Text>
         )}
-        {err && (
-          <Label color='status.fail.background' mt={3} mb={0}>
-            {err}
-          </Label>
-        )}
+        <Label color='status.fail.background' minHeight={6} mt={3} mb={0}>
```

Comment:
We only want to render this if there's an `err` right?

Reply:
Good catch, but no. By moving the conditional rendering to the error message content - rather than the component as a whole - we avoid the page "jumping" slightly as the component is inserted into the DOM and bumping the other elements above it "up".